src/CTA.Rules.Metrics/PortSolutionResultReportGenerator.cs

```diff
@@ -140,6 +149,11 @@ private string GeneratePortSolutionResultJsonReport()
                 .Concat(MissingMetaReferenceMetrics)
                 .ToList();
 
+            if (WebFormsActionMetrics != null)
+            {
+                allMetrics = allMetrics.Concat(WebFormsActionMetrics).ToList();
+            }
```

Comment:
Similar to above, if we always initialize this to an empty collection, we can remove the if statement and always concat it.